src/palindrom.js

```diff
@@ -823,6 +765,10 @@ var Palindrom = (function () {
 
   /* backward compatibility */
   global.Puppet = Palindrom;
+  
+  /* Since we have jsonpatch bundled,
+  let's expose it in case anyone needs it */
+  global.jsonpatch = jsonpatch;
```

Comment:
@warpech because Angularjs demo uses it to apply changes to `$scope`. And I thought why not?